packages/core/src/utils/shell/copyAll.ts

```diff
@@ -37,5 +37,5 @@ export const copyAll = async (src: string, dest: string, prompt: boolean = true,
 		shell.mkdir('-p', dest);
 	}
 
-	shell.cp('-rf', src + '/*', dest);
+	shell.cp('-rf', src + '/.*', dest);
```

Comment:
its tested?

Reply:
tested the regex with certain file name but not with the local cli

Reply:
can you please do that an confirm. we can then go on to merge this

Reply:
Tested it.